styles/index.css

```
@@ -142,4 +142,12 @@ body.modal-active {
     opacity: .96;
 }
 
```

When using my menu-item suggestion, we need to add this class to the styles, since it seems not to be available in the standard library.
```suggestion

.pt-7 {
    padding-top: 1.75rem;
}

```

---

I'm not in favor of adding custom style classes, this should be a last resort only. Working with 8+4 instead of 7+5 split works as well.